functorch/_src/eager_transforms.py

```diff
@@ -249,13 +249,10 @@ def jvp(f, primals, tangents):
         JVP_NESTING += 1
         ctx = fwAD.dual_level if JVP_NESTING == 1 else noop
         with ctx():
-            # TODO: extend this to any number of primals
-            assert len(primals) == 1 and len(tangents) == 1
             duals = tuple(fwAD.make_dual(p, t) for p, t in zip(primals, tangents))
             result_duals = f(*duals)
             result_duals, _ = tree_flatten(result_duals)
-            assert len(result_duals) == 1
-            primals_out, tangents_out = fwAD.unpack_dual(result_duals[0])
+            primals_out, tangents_out = zip(*(fwAD.unpack_dual(d) for d in result_duals))
```

Comment:
You should pull main and use the changes from main; I did the exact same thing here and added an error check